tests/baselines/reference/regularExpressionAnnexB.errors.txt

```diff
@@ -0,0 +1,272 @@
+regularExpressionAnnexB.ts(32,7): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(33,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(33,7): error TS1005: '}' expected.
+regularExpressionAnnexB.ts(33,8): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(34,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(34,8): error TS1005: '}' expected.
+regularExpressionAnnexB.ts(34,9): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(35,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(35,5): error TS1506: Numbers out of order in quantifier.
+regularExpressionAnnexB.ts(35,8): error TS1005: '}' expected.
+regularExpressionAnnexB.ts(35,9): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(36,4): error TS1508: Unexpected '{'. Did you mean to escape it with backslash?
+regularExpressionAnnexB.ts(36,5): error TS1508: Unexpected '}'. Did you mean to escape it with backslash?
+regularExpressionAnnexB.ts(37,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(37,5): error TS1505: Incomplete quantifier. Digit expected.
+regularExpressionAnnexB.ts(37,8): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(38,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(38,5): error TS1505: Incomplete quantifier. Digit expected.
+regularExpressionAnnexB.ts(38,9): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(39,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(39,8): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(40,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(40,9): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(41,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(41,10): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(42,4): error TS1507: There is nothing available for repetition.
+regularExpressionAnnexB.ts(42,5): error TS1506: Numbers out of order in quantifier.
+regularExpressionAnnexB.ts(42,10): error TS1507: There is nothing available for repetition.
+
+
+==== regularExpressionAnnexB.ts (75 errors) ====
+    const regexes: RegExp[] = [
+      /\q\u\i\c\k\_\f\o\x\-\j\u\m\p\s/,
+           
+!!! error TS1125: Hexadecimal digit expected.
+               ~~
+!!! error TS1510: '\k' must be followed by a capturing group name enclosed in angle brackets.
+                         
+!!! error TS1125: Hexadecimal digit expected.
+                               
+!!! error TS1125: Hexadecimal digit expected.

```

Comment:
These are all valid in Annex B. Even `/\u{1/` is. In Annex B essentially all weird things are valid, you know. Currently (after #58295) `/[\1]/` is valid but `/[\8]/` isn’t. I don’t think it’s ideal.